### ISSUE TITLE
SDS scoring model

### ISSUE THREAD

**Issue by mcfrank** [levante-framework/levante-pilots#12]
Same different selection is currently scoring item types, which seems reasonable. Need to check if we can consolidate these more and compare different model variants. Perhaps we can prune some items.

**Comment by mikabr** [comment]
@mcfrank what do you mean by consolidate more? by compare model variants do you mean compare treating each item type as an item vs. each individual item instance as in item?

**Comment by mikabr** [comment]
TODO: create score for test dimensions, exclude kids from rest of task if they're below some threshold for test dimensions

**Comment by mcfrank** [comment]
Yes - I chose one grouping of item types but we migh consider which
grouping choice leads to highest reliability. For example we could treat
every item separately (which is worse I think) or lump all items in a level
together.
